sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj

```diff
@@ -58,4 +58,10 @@
       <CustomToolNamespace>Azure.Messaging.EventHubs</CustomToolNamespace>
     </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup>
+    <None Remove="W:\home\jesse\src\microsoft\azure-sdk-dotnet\artifacts\obj\Azure.Messaging.EventHubs\Debug\netstandard2.0\README.md" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Remove="W:\home\jesse\src\microsoft\azure-sdk-dotnet\artifacts\obj\Azure.Messaging.EventHubs\Debug\netstandard2.0\README.md" />
```

Comment:
Remove?

Reply:
Again. odd.   I'll get those fixed up when I push the API gen code that I forgot.